package.json

```
@@ -2,9 +2,10 @@
 	"name": "node.js_course_practice",
 	"version": "1.0.0",
 	"description": "",
-	"main": "app.js",
+	"main": "./dist/app.js",
 	"scripts": {
-		"start": "node app.js",
+		"build": "tsc",
+		"start": "npm run build && node ./dist/app.js",
```

Let's separate start command that used commonly to run production build and command for development

```suggestion
		"start": "npm run build && node .",
		"start:dev": "nodemon src/app.ts",
```

---

Have added start:dev